engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/error/displaytext/ForbiddenOperationToDisplayTextNode.java

```diff
@@ -0,0 +1,31 @@
+package org.enso.interpreter.node.expression.builtin.error.displaytext;
+
+import com.oracle.truffle.api.dsl.Specialization;
+import com.oracle.truffle.api.library.CachedLibrary;
+import com.oracle.truffle.api.nodes.Node;
+import org.enso.interpreter.dsl.BuiltinMethod;
+import org.enso.interpreter.runtime.callable.atom.Atom;
+import org.enso.interpreter.runtime.callable.atom.AtomConstructor;
+import org.enso.interpreter.runtime.callable.atom.StructsLibrary;
+import org.enso.interpreter.runtime.data.text.Text;
+
+@BuiltinMethod(type = "Forbidden_Operation", name = "to_display_text")
+public abstract class ForbiddenOperationToDisplayTextNode extends Node {
+  static ForbiddenOperationToDisplayTextNode build() {
+    return ForbiddenOperationToDisplayTextNodeGen.create();
+  }
+
+  abstract Text execute(Object self);
+
+  @Specialization
+  Text doAtom(Atom self, @CachedLibrary(limit = "3") StructsLibrary structs) {
```

Comment:
I know it is quite common to have these `XyzToDisplayTextNode` implementation spread around the code base, but is there are reason for that?

Why can't we simply write following?
```
Forbidden_Operation.to_display_text self = "Forbidden operation: "+self.why+"."
```
CCing @jdunkerley as he asked this question once.